lettuce/reporters.py

```diff
@@ -140,7 +147,13 @@ def __call__(self, i, t, f):
             else:
                 observed = observed.tolist()
             entry = [i, t] + observed
-            if isinstance(self.out, list):
+            if self.save_to_file:
                 self.out.append(entry)
-            else:
-                print(*entry, file=self.out)
\ No newline at end of file
+            if self.print_to_screen:
+                print(*entry)  # , file=self.out)
+
+    def save(self):
+        if os.path.exists(self.filename_base):
+            self.filename_base += datetime.now().strftime('_%Y%m%d_%H%M%S')  # "_" + str(datetime.now()).replace(".", "").replace(":", "").replace(" ", "").replace("-", "")
```

Comment:
If you really need this, why not make a custom observable "Datetime" and log this, too?